.github/workflows/tkg_integration_tests.yaml

```diff
@@ -96,6 +96,10 @@ jobs:
     - name: Cleanup
       run: rm -rf ~/.tanzu ~/.tkg ~/.config; docker kill $(docker ps -q) | true; docker system prune -a --volumes -f
 
+    - name: Build CLI
```

Comment:
What are this section and below one for?

Reply:
Because these two workflows are defined for CAPD and CAPZ, they are both for pure legacy integration tests. So we need to hack use below commands:
```
tanzu config set features.management-cluster.package-based-cc false
tanzu config set features.cluster.allow-legacy-cluster true
```
Then we need these two parts to install tanzu cli, otherwise, it will complain `tanzu command not found` during run hack.